code/game/objects/items/weapons/material/material_weapons.dm

```diff
@@ -2,13 +2,15 @@
 // This class of weapons takes force and appearance data from a material datum.
 // They are also fragile based on material data and many can break/smash apart.
 /obj/item/material
-	hitsound = 'sounds/weapons/bladeslice.ogg'
 	gender = NEUTER
 	throw_speed = 3
 	throw_range = 7
 	w_class = ITEM_SIZE_NORMAL
 	health_max = 10
 
+	abstract_type = /obj/item/material
+
```

Comment:
Whats with the spaces?

Reply:
also, abstract_type should be right underneath the type itself